app/components/daedalusTransfer/DaedalusTransferErrorPage.js

```diff
@@ -28,6 +28,9 @@ type Props = {
 
 @observer
 export default class DaedalusTransferErrorPage extends Component<Props> {
+  static defaultProps = {
```

Comment:
where is this used? I didn’t see it

Reply:
it's used by React internally to give a default value for the arguments to the React component

Reply:
okay